tensorflow/lite/kernels/activations.cc

```diff
@@ -76,9 +76,10 @@ struct LogSoftmaxOpData : public OpData {
 };
 
 struct LeakyReluOpData : public OpData {
-  uint8_t q_alpha;
-  int32_t output_multiplier = 0;
-  int output_shift = 0;
+  int32_t output_multiplier_alpha = 0;
```

Comment:
can you either use int or int32_t for consistency? thanks

Reply:
No problem. All switched to int32_t. Just a note, `MultiplyByQuantizedMultiplier` and `QuantizeMultiplier` use mixture of int and int32(_t), not sure if it is intended.